### PR TITLE
Add compat data for SVGAElement

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/SVGAElement

### CODE REVIEW

**Comment by teoli2003** [comment]
I'm not sure if it is a good idea to do this now, at the beginning of a reorganization. It may be, I would like @elchi3 and @JeremiePat opinion here.

**Comment by maboa** [comment]
Happy to work on other APIs @teoli2003 - any suggestions for what to look at next? Thanks!

**Comment by JeremiePat** [comment]
Hi!

[I plan to add all SVG compat data](https://discourse.mozilla.org/t/time-to-lift-up-the-svg-documentation-to-the-next-level/25529) in March during the Hack On MDN event. But it's fine to start right now :)

Especially for the API part as [it exist a bit more than 170 APIs](https://docs.google.com/spreadsheets/d/17azdgEZUsAqmYqaQDvhjH90Q_kv07mzxLD4Htkqf12c/edit?usp=sharing).

For what it worth, the reorganization part should not impact the API (only Elements and Attributes) except some page refresh which totally independent than the BCD work.

I'll do a quick review of the PR as some information needs to be updated.

File: api/SVGAElement.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGAElement",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/SVGAElement.json]: Should be `true`.

File: api/SVGAElement.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          "description": "SVGAElement.target",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SVGAElement.json]: Should be `true`.

File: api/SVGAElement.json (hunk 1 above)
Comment (maboa) [reply api/SVGAElement.json]: 👍

File: api/SVGAElement.json (hunk 2 above)
Comment (maboa) [reply api/SVGAElement.json]: 👍

File: api/SVGAElement.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+            },
+            "edge": {
+              "version_added": null
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/SVGAElement.json]: 👍

File: api/SVGAElement.json [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            },
+            "edge_mobile": {
+              "version_added": null
```
[the remark this answers is not in the document]
Comment (maboa) [reply api/SVGAElement.json]: 👍

File: api/SVGAElement.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (maboa) [reply api/SVGAElement.json]: 👍

File: api/SVGAElement.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (maboa) [reply api/SVGAElement.json]: 👍

**Comment by maboa** [comment]
I've moved on to IndexDB in the meantime, but may come back to SVG. 

Unfortunately I won't be able to make the meet up in Paris in March :(

File: api/SVGAElement.json [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "target": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGAElement/SVGAlement.target",
```
Comment (Elchi3) [inline api/SVGAElement.json]: https://developer.mozilla.org/docs/Web/API/SVGAElement/target is the correct URL. It seems like there is a duplicate page on MDN which I will redirect.

File: api/SVGAElement.json [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+          "description": "SVGAElement.target",
```
Comment (Elchi3) [inline api/SVGAElement.json]: description not needed

**Pull request merged**